sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -1,3090 +1,3131 @@
-          "connectedDeviceCount": {
-            "format": "int64",
-            "type": "integer"
-          }
+      }
+    },
+    "ServiceStatistics": {
+      "type": "object",
+      "properties": {
+        "connectedDeviceCount": {
+          "format": "int64",
+          "type": "integer",
+          "description": "The number of currently connected devices."
         }
-      },
-      "Device": {
-        "type": "object",
-        "properties": {
-          "deviceId": {
-            "type": "string"
-          },
-          "generationId": {
-            "type": "string"
-          },
-          "etag": {
-            "type": "string"
-          },
-          "connectionState": {
-            "enum": [
-              "Disconnected",
-              "Connected"
-            ],
-            "type": "string"
-          },
-          "status": {
-            "enum": [
-              "enabled",
-              "disabled"
-            ],
-            "type": "string"
-          },
-          "statusReason": {
-            "type": "string"
-          },
-          "connectionStateUpdatedTime": {
-            "format": "date-time",
-            "type": "string"
-          },
-          "statusUpdatedTime": {
-            "format": "date-time",
-            "type": "string"
-          },
-          "lastActivityTime": {
-            "format": "date-time",
-            "type": "string"
-          },
-          "cloudToDeviceMessageCount": {
-            "format": "int32",
-            "type": "integer"
-          },
-          "authentication": {
-            "$ref": "#/definitions/AuthenticationMechanism"
-          },
-          "capabilities": {
-            "$ref": "#/definitions/DeviceCapabilities"
-          },
-          "deviceScope": {
+      }
+    },
+    "Device": {
+      "type": "object",
+      "properties": {
+        "deviceId": {
+          "type": "string",
+          "description": "The unique identifier of this device."
+        },
+        "generationId": {
+          "type": "string",
+          "description": "An IoT hub-generated, case-sensitive string up to 128 characters long. This value is used to distinguish devices with the same deviceId, when they have been deleted and re-created."
+        },
+        "etag": {
+          "type": "string",
+          "description": "A string representing a weak ETag for the device identity, as per RFC7232."
+        },
+        "connectionState": {
+          "enum": [
+            "Disconnected",
+            "Connected"
+          ],
+          "type": "string",
+          "description": "Tells whether the device is connected or not."
+        },
+        "status": {
+          "enum": [
+            "enabled",
+            "disabled"
+          ],
+          "type": "string",
+          "description": "An access indicator. Can be Enabled or Disabled. If Enabled, the device is allowed to connect. If Disabled, this device cannot access any device-facing endpoint."
```

Comment:
What do you mean by this:
If Disabled, this device cannot access any **device-facing endpoint**.?

Reply:
This was from the official iothub doc, by I agree it is confusing. I'll change this